try {
	1 / 0
} catch (Exception e){
	println e
}